frontend/types/api/notifications.ts

```diff
@@ -43,7 +43,7 @@ export interface NotificationDashboardsTableRow {
   group_name: string;
   notification_id: number /* uint64 */; // may be string? db schema is not defined afaik
   entity_count: number /* uint64 */;
-  event_types: string[];
+  event_types: 'validator_online' | 'validator_offline' | 'group_online' | 'group_offline' | 'attestation_missed' | 'proposal_success' | 'proposal_missed' | 'proposal_upcoming' | 'sync' | 'withdrawal' | 'got_slashed' | 'has_slashed' | 'incoming_tx' | 'outgoing_tx' | 'transfer_erc20' | 'transfer_erc721' | 'transfer_erc1155';
```

Comment:
I think this should be a string.

Reply:
Not sure if I understand correctly, the point of this was to not use a `string` but a union type, right? But I see that it's not an array of event types any more, just a single value - that wasn't intended, changed it (thanks!). Was this was you meant or should I change it to something else?

Reply:
Yes exactly that, thanks ;-)